lib/widgets/profile.dart

```diff
@@ -33,6 +35,37 @@ class ProfilePage extends StatelessWidget {
       page: ProfilePage(userId: userId));
   }
 
+  /// The given user's real email address, if known, for displaying in the UI.
+  ///
+  /// Returns null if self-user isn't able to see [user]'s real email address.
+  ///
+  /// **Note:** Starting from Zulip version 7.0 (FL 163), there's a change in
+  /// the API about how self-user can access [user]'s real email address.
+  /// Search for "delivery_email" in https://zulip.com/api/register-queue.
+  // TODO(server-7, FL >= 163): remove
```

Comment:
```suggestion
  // TODO(server-7): remove; use user-level setting
```

(The form we've settled on is `TODO(server-x)`; it doesn't need the feature level. All servers that are 7+ will have FL >= 163.)